README.md

```diff
@@ -46,7 +46,7 @@ source ~/.local/share/virtualenvs/driloader-ZPASDPNJ/bin/activate
 from driloader import driloader
 from selenium.webdriver import Firefox
 
```

Comment:
It seems to me that your changes break the compatibility of this project with Python Versions under 3.6. Please take a look the build https://travis-ci.org/github/lucasmello/Driloader/jobs/693222868.

If this is true, the README must be updated as well as the `setup.py`.

Reply:
I also noticed that the `make lint` is breacking:
```
$ make lint 
pylint -d fixme driloader
Using config file /media/jonatha/Storage/jonatha/Documents/Projetos/Python/driloader/pylintrc
************* Module driloader.browser.firefox
E:  1, 0: Bad option value 'import-outside-toplevel' (bad-option-value)
************* Module driloader.utils.file
E:  1, 0: Bad option value 'import-outside-toplevel' (bad-option-value)

------------------------------------------------------------------
Your code has been rated at 9.81/10 (previous run: 9.81/10, +0.00)

Makefile:21: recipe for target 'lint' failed
make: *** [lint] Error 2
```